features/config.feature

```diff
@@ -35,3 +35,59 @@ Feature: Config
 
           When this scenario executes
       """
+
+  Scenario: Alternative configuration file
+    Given a file named "alternative-behat.yml" with:
+    """
```

Comment:
Indentation of your opening `"""` is off in all scenarios (this is common when copy/pasting)